src/Debugger/Daemon.php

```diff
@@ -54,23 +53,51 @@ class Daemon
      */
     private $storage;
 
+    /**
+     * @var array Configuration options for setting up the default DebuggerClient.
+     */
+    private $clientOptions;
+
+    /**
+     * @var array Source context configuration.
+     */
+    private $extSourceContext;
+
+    /**
+     * @var string The uniquifier for this daemon's debuggee.
+     */
+    private $uniquifier;
+
+    /**
+     * @var string The description for this daemon's debuggee.
+     */
+    private $description;
+
+    /**
+     * @var array A set of custom debuggee properties, populated by the agent,
+     *      to be displayed to the user.
+     */
+    private $labels;
+
     /**
      * Creates a new Daemon instance.
      *
-     * @param string $sourceRoot The full path to the source root
      * @param array $options [optional] {
      *      Configuration options.
      *
-     *      @type DebuggerClient $client A DebuggerClient to use. **Defaults
-     *            to** a new DebuggerClient.
-     *      @type array $extSourceContext The source code identifier. **Defaults
+     *      @type string $sourceRoot The full path to the source root
+     *      @type array $clientOptions The options to instantiate the default
```

Comment:
Do you think it would be handy to link out to the constructor for the debugger client here, in order to quickly look at options?

Reply:
Sure, will do.